catkit/hardware/iris_ao/iris_ao_controller.py

```diff
@@ -117,14 +113,12 @@ def zero(self, return_zeros=False):
         :return: If return_zeros=True, return a dictionary of zeros
         """
         zero_list = util.create_zero_list(util.iris_num_segments(self.config_id))
-        zeros = util.create_dict_from_list(zero_list, util.iris_pupil_naming(self.config_id))
-        self.send_data(zeros)
-
-        # Update the testbed state
-        self._update_iris_state(zeros)
+        dm_shape = SegmentedDmCommand(dm_config_id=self.config_id)
+        dm_shape.read_initial_command(zero_list)
+        self.apply_shape(dm_shape)
```

Comment:
I need to test this on hardware.

@kjbrooks, @ivalaginja does this look correct at least, I wasn't sure?

Reply:
Yes, this looks fine to me (as long as `self` is an instance of some sort of iris ao object).